src/wmnds/patterns/content-cards/travel-updates/_travel-updates.njk

```diff
@@ -0,0 +1,60 @@
+{% macro wmndsContentCardTravelUpdates(params) %}
+{# imports #}
+{% from "wmnds/components/content-card/_content-card.njk" import wmndsContentCard %}
+{% from "wmnds/components/disruption-indicator/large/_large.njk" import wmndsDisruptionIndicatorLarge as largeDisruption %}
+{% from "wmnds/components/disruption-indicator/medium/_medium.njk" import wmndsDisruptionIndicatorMedium as mediumDisruption %}
+{% from "wmnds/components/message/help/_help.njk" import wmndsMsgHelp %}
+{# Set #}
+{% set largeDisruptionsAreas = ["Hagley Road", "Ring Road"] %}
+{% set buses = ["9", "10", "12", "12A", "13", "13A", "126", "X8", "X10", "23"] %}
+
+{% call wmndsContentCard({ variant: "wmnds-content-card--travel-updates"}) %}
+    <div class="wmnds-p-sm wmnds-col-1 wmnds-col-md-10-24">
```

Comment:
```suggestion
    <div class="wmnds-p-sm wmnds-col-1">
```

I would remove this class as it causes problems around the 820px screen width range.
As we don't know how this will be used (in a sidebar, in the main content area, full page?) it's probably best to make it full width so it scales better for all use cases.

I know it doesn't look exactly like the design(s) but if you put it in a sidebar it will more or less look the same as it will scale to fit.